tests/test_options.py

```diff
@@ -39,8 +39,8 @@ def test_official_mda_theme(self, tmpdir):
 
             text = conf.read_text()
             assert '"mda_official": True,' in text
-            assert "html_logo" not in text
-            assert "html_favicon" not in text
+            # assert "html_logo" not in text
```

Comment:
maybe worth removing these if they are no longer useful?

Reply:
Thank you -- done!